packages/cli/src/cli/commands/sites.ts

```diff
@@ -16,6 +18,39 @@ export const createSite = errorHandler<string>(async (url: string) => {
   }
 });
 
+type getComponentSchemaParams = { url: string; apiPath: string | null };
+export const getComponentSchema = errorHandler<getComponentSchemaParams>(
+  async ({ url, apiPath }: getComponentSchemaParams) => {
+    const spinner = ora("Creating site...").start();
+    try {
```

Comment:
If its to handle unexpected errors, you can remove it.
`errorHandler` will take care of it and also display stac trace.